src/clusterfuzz/_internal/tests/core/bot/fuzzers/afl/afl_launcher_integration_test.py

```diff
@@ -48,7 +48,7 @@ def create_temp_dir():
 
```

Comment:
Do the existing tests test this functionality you added?

Reply:
I guess not.